heroic-core/src/main/java/com/spotify/heroic/HeroicCore.java

```diff
@@ -312,6 +312,11 @@ private void enableTracing(Map<String, Object> config) throws IOException {
                 DEFAULT_LIGHTSTEP_REPORTING_MS);
             final Integer maxBufferedSpans = (Integer) config.getOrDefault("maxBufferedSpans",
                 DEFAULT_LIGHTSTEP_MAX_SPANS);
+            final Boolean grpcRoundRobin = (Boolean) config.getOrDefault("grpcRoundRobin",
+                DEFAULT_GRPC_ROUNDROBIN);
+            final Boolean grpcResetClient = (Boolean) config.getOrDefault("grpcResetClient",
+                DEFAULT_GRPC_RESETCLIENT);
+            final String grpcCollectorTarget = configMap.get("grpcCollectorTarget");
 
 
             if (collectorHost == null || accessToken == null) {
```

Comment:
if `grpcCollectorTarget` is required with no default, we should add a null check for it here